adapter/src/adapter/importer/HttpImporter.ts

```diff
@@ -0,0 +1,86 @@
+import { ImporterParameterError } from "../model/exceptions/ImporterParameterError";
+import { Importer } from "./Importer";
+import { ImporterParameterDescription } from "./ImporterParameterDescription";
+const axios = require('axios');
+
+export class HttpImporter extends Importer {
+
+  //TODO RuntimeParameters type is probably wrong
+  parameters:ImporterParameterDescription[] = [new ImporterParameterDescription({name:"location", description:"String of the URI for the HTTP call", type:"string"}), 
```

Comment:
Alternative design idea:

Use an interface instead of a class for parameter descriptions (since no behavior, just plain data)
```TypeScript```
export interface ImporterParamDescription {
  name: string;
  description: string;
  type: string;
}
```

Use a record ("as a map"). Not sure if we need this though.
```TypeScript
parameters: Record<string, ImporterParamDescription> = {
  'location': {
    name: 'location',
    description: '....',
    type: 'string'
  }
}

```